src/client/pydaos/pydaos_core.py

```diff
@@ -124,13 +136,16 @@ def dict(self, name, v: dict = None):
 
         return dd
 
-    def array(self, name, v: list = None):
+    def array(self, name, v: list = None, cid="0"):
         # pylint: disable=unused-argument
         """ Create new DArray object """
 
+        # Get the existing class ID based on class name given. Default to 0
+        objId = _get_object_id(cid)
```

Comment:
just to verify this works if cid == 0?
is objID in this case (0)?
could be something like:
if (cid != "0")
  objId = _get_object_id(cid)
else
  objId = 0

Reply:
That check is in _get_object_id() function above.